pkg/cinder/initcontainer.go

```diff
@@ -94,6 +94,22 @@ func InitContainer(init APIDetails) []corev1.Container {
 			},
 		},
 	}
+
+	if init.TransportURLSecret != "" {
```

Comment:
?: Wondering if we should fail if we don't have the transport url secret...

Reply:
Well, this init container code is currently shared between the various CRs, so that's why the check for `TransportURLSecret` is here (only `Cinder` will use it, while the others will not).  The actual check for the secret itself is handled here: https://github.com/abays/cinder-operator/blob/56ff77d5af054f4fd667876734e5ad347ed001ac/controllers/cinder_controller.go#L380-L388